lua/entities/ent_jack_gmod_ezblackhole.lua

```diff
@@ -29,7 +31,13 @@ function ENT:SUCC(Time,Phys,Age,Pos,MaxRange)
 			end
 		elseif(table.HasValue(self.RagdollifyEnts,Class))then
 			if((SERVER)and(math.random(1,100)==42))then obj:Fire("becomeragdoll","",0) end
-		elseif((IsValid(ObjPhys))and not(obj:IsWorld())and not(obj==self)and not(self:IsBlacklisted(obj)))then
+		elseif table.HasValue(self.DamageEnts, Class) then
+			local Vec=Pos-obj:GetPos()
+			local Dist,Dir=Vec:Length(),Vec:GetNormalized()
+			if SERVER and Dist/MaxRange < 0.1 then
```

Comment:
i'm not sure about this, i'd like the damage to scale based on distance/lifetime but i'll deal with that after the merge

Reply:
I'll submit a commit with a simple scaling. Might not work too well considering breakables have different health values, but I don't believe it's possible to scale based off prop max hp.